wdl2cwl/tests/cwl_files/NoResult.cwl

```diff
@@ -0,0 +1,29 @@
+class: CommandLineTool
+id: CollectQualityYieldMetrics
+inputs:
+- id: input_bam
+  type: File
+- id: metrics_filename
+  type: string
+- id: preemptible_tries
+  type: int
+outputs:
+- id: quality_yield_metrics
+  type: File
+  outputBinding:
+    glob: $(inputs.metrics_filename)
+requirements:
+- class: DockerRequirement
+  dockerPull: us.gcr.io/broad-gotc-prod/picard-cloud:2.23.8
+- class: InitialWorkDirRequirement
+  listing:
+    entryname: example.sh
+    entry: '    java -Xms2000m -jar /usr/picard/picard.jar \      CollectQualityYieldMetrics
+      \      INPUT=$(inputs.input_bam.path)\      OQ=true \      OUTPUT=$(inputs.metrics_filename)  '
```

Comment:
You should split the `command` string on `\` to break up the lines, and then trim off the trailing `\`